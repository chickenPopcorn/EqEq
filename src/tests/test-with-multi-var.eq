SomeCtx = {
  a = { 42; }
  b = { 32; }
  c = { 22; }

}

SomeCtx:find a with b =17; c = 7; {
  print("%0.0f\n", a);
  print("%0.0f\n", b);
  print("%0.0f\n", c);
}
