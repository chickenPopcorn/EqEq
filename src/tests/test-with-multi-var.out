42
17
7
